### ISSUE TITLE
Not Found in HACS

### ISSUE THREAD

**Issue by JAAlperin** [danobot/entity-controller#189]
<<< Do not delete the following template headers. All information must be filled in because we need it to analyse the problem. Your issue will be auto closed without this information.
Before opening a new issue, please read the documentation and search for keywords relating to your problem. Since this is an open source project expect to be involved in the issue resolution because we depend on your support. >>>

## Description
"Entity Controller" is not found in HACS Integrations

## Configuration
HACS 1.5.2 on Home Assistant 0.114.4.

## Steps to reproduce
Follow Installation instructions in github

## Logs
Provide logs at DEBUG level. 

## Version
If you are not using the latest version, please upgrade first.

**Comment by danobot** [comment]
Its visible for me in HACS

**Comment by danobot** [comment]
Try reloading HACS data

**Comment by JAAlperin** [comment]
Hello Daniel.

I have reloaded HACS data - no change.  I reinstalled HACS - no change (search for “entity” in integrations produces “no repository”).  However I can add your repository (which should not be possible if your repository were included in HACS).  I don’t doubt that it works for you, but it does not for me.  So, there is definitely something wrong.  I can see no other problems with HACS on my end.  New repositories and updates show up often.

I am running HASS 0.114.4 Sys 243 in docker 19.03.12 on Ubuntu 20.04.1 LTS on a Lenovo M93p.  I also run HASS and HACS on some pi4s.  They cannot find your integration either.

I look forward to installing your integration, and thank you for the contribution to Home Assistant.

Best Regards,

Jeff Alperin

H: 561-666-7789
C: 561-797-9876
